packages/flutter_tools/lib/src/commands/build_ios.dart

```diff
@@ -724,6 +725,21 @@ abstract class _BuildIOSSubCommand extends BuildSubCommand {
     if (result.output != null) {
       globals.printStatus('Built ${result.output}.');
 
+      // When an app is successfully built, record to analytics whether Impeller
+      // is enabled or disabled.
+      final BuildableIOSApp app = await buildableIOSApp;
+      final String plistPath = app.project.infoPlist.path;
+      final bool? impellerEnabled = globals.plistParser.getValueFromFile<bool>(
+        plistPath, PlistParser.kFLTEnableImpellerKey,
+      );
+      BuildEvent(
+        impellerEnabled == false
```

Comment:
so in the absence of this key impeller is used?

Reply:
the flag is impeller enabled but the setting is impeller disabled :)